Makefile

```diff
@@ -1,8 +1,21 @@
 init::
+	python -m pip install --upgrade pip
 	python -m pip install pip-tools
-	python -m piptools sync task/dev-requirements.txt
+	python -m piptools compile task/requirements/dev-requirements.in
```

Comment:
Nervous about compiling on init as it may update all the packages and break stuff not sure if recompiling on demand is better. Admittedly we may fall behind on the packages we're using

Reply:
No problem I'll switch back to how it was done before